tests/e2e/Services/Migrations/MigrationsBase.php

```
@@ -1694,4 +1694,839 @@ public function testCreateCSVExport(): void
+
+        $subscriber = $this->client->call(Client::METHOD_POST, '/messaging/topics/' . $topicId . '/subscribers', [
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey'],
+        ], [
+            'subscriberId' => ID::unique(),
+            'targetId' => $targetId,
+        ]);
+
+        $this->assertEquals(201, $subscriber['headers']['status-code']);
+        $subscriberId = $subscriber['body']['$id'];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$subscriberId'. (undefined)

(UnusedLocalVariable)